docs/source/standard.md

```diff
@@ -113,65 +113,130 @@ The ability of the TRE organisation to provide and manage devices, workspaces, i
 
 #### 2.1.1 User interface
 
-terminal, desktop, notebook, webapp etc.
-
-| Statement | Guidance |
-| --------- | -------- |
-|           |          |
+The interfaces used for interacting with the TRE management system and the TRE workspace.
+
+```{list-table}
+:header-rows: 1
+:name: tab-end-user-user-interface
+* - Statement
+  - Guidance
+* - A TRE must be accessed via a user interface accessible using commonly available applications.
+  - TREs should not require the installation of any bespoke TRE application on the user's device. In practice a web browser is the most common way to achieve this.
```

Comment:
Maybe we should add the caveat "If users connect with their own/own organisation's devices"?

If a TRE provided managed devices, or access was only through a dedicated machine in a secure room I would be less fussed if it involved bespoke tool.s.